src/ui/screens/settings/privacy/ViewPrivacy.qml

```diff
@@ -106,5 +125,26 @@ MZViewBase {
 
         onActiveChanged: if (active) { item.open() }
     }
+
+    Loader {
+        id: helpSheetLoader
+
+        active: false
+
+        onActiveChanged: if (active) item.open()
+
+        sourceComponent: MZHelpSheet {
+            title: MZI18n.HelpSheetsPrivacyTitle
+
+            model: [
+                {type: MZHelpSheet.BlockType.Title, text: MZI18n.HelpSheetsPrivacyHeader},
+                {type: MZHelpSheet.BlockType.Text, text: MZI18n.HelpSheetsPrivacyBody1, margin: 8},
+                {type: MZHelpSheet.BlockType.Text, text:MZI18n.HelpSheetsPrivacyBody2, margin: 16},
+                {type: MZHelpSheet.BlockType.LinkButton, text: MZI18n.GlobalLearnMore, margin: 16, action: () => { MZUrlOpener.openUrlLabel("sumoPrivacy") } },
+            ]
+
+            onClosed: helpSheetLoader.active = false
```

Comment:
Do we anticipate these help sheets existing outside of Loaders? Any reason to not create a separate component for the Loader/HelpSheet duo? Seems like we could save ourselves a bit of duping.

Reply:
agree. [see here](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8873#discussion_r1457908929)